docs/storage/request_queue.md

```diff
@@ -1,50 +1,219 @@
-    await queue.addRequest({ url: 'http://example.com/bbb' });
-    await queue.addRequest({ url: 'http://example.com/foo/bar' }, { forefront: true });
-
-    // Get requests from queue
+    await queue.addRequest(
+        { url: 'http://example.com/foo/bar' },
+        { forefront: true }
+    );
+    
+    // Get the next request from queue
     const request1 = await queue.fetchNextRequest();
     const request2 = await queue.fetchNextRequest();
-    const request3 = await queue.fetchNextRequest();
 
-    // Mark a request as handled
-    await queue.markRequestHandled(request1);
+    // Get a specific request
+    const request22 = await queue.getRequest('request-22');
 
-    // If the processing of a request fails then reclaim it back to the queue, so that it's crawled again
+    // Reclaim a failed request back to the queue
+    // and crawl it again
     await queue.reclaimRequest(request2);
 
-## [](#api-and-javascript-client)API and Javascript client
+    // Remove a queue
+    await queue.drop();
+
+For more information on managing your request queues with the Apify SDK, see the SDK [documentation](https://sdk.apify.com/docs/guides/data-storage#request-queue) and [API reference](https://sdk.apify.com/docs/api/request-queue).
+
+### JavaScript API client
+
+Apify's [JavaScript API client](https://docs.apify.com/apify-client-js#ApifyClient-requestQueues) (`apify-client`) allows you to access your request queues from outside the Apify platform (e.g. from a Node.js application).
+
+For help with setting up the JavaScript API client, see the Storage documentation's [overview page](https://docs.apify.com/storage/#javascript-api-client).
+
+After importing the `apify-client` package into your application and creating an instance of it, save it to a variable for easier access.
+
+    // Save your request queues to a variable for easier access
+    const requestQueues = apifyClient.requestQueues;
+
+You can then get or create new queues, retrieve existing requests or enqueue new URLs. For a full list of options, see the [JavaScript API client's](https://docs.apify.com/apify-client-js#ApifyClient-requestQueues) documentation.
+
+    // Get the 'my-queue' request queue and set it as the default
+    // to be used in the following commands
+    const queue = await requestQueues.getOrCreateQueue({
+        queueName: 'my-queue',
+    });
+    apifyClient.setOptions({ queueId: queue.id });
+
+    // Add a request to the default queue
+    await requestQueues.addRequest({ 
+        url: 'http://example.com',
+        uniqueKey: 'http://example.com'
+    });
+
+    // Add a request to 'my-queue'
+    await requestQueues.addRequest(
+        queueId='my-queue',
+        {
+            url: 'http://example.com/a/b',
+            uniqueKey: 'http://example.com/a/b', 
+        }
+    );
+
+    // Fetch unhandled requets from queue.
+    const [request1, request2] = await requestQueues.queryQueueHead();
```

Comment:
`queryQueueHead` method does not exist.

Reply:
Good thing API client docs are getting updated then :)